### DISCUSSION TITLE
Can not use mikro-orm.config file as commonjs module

### DISCUSSION THREAD

**Discussion by yaroslavyal** [mikro-orm/mikro-orm#2780] [Q&A]
**Describe the bug**
After upgrading to the 5v can't use compiled mikro-orm.config.js file as commonjs module. I found in previous version was used `require` for loading [config](https://github.com/mikro-orm/mikro-orm/blob/31ab48b8e49625108697c5e281b7b8c58afb09c1/packages/core/src/utils/ConfigurationLoader.ts#L21). But in the current version is used `import` (this [line](https://github.com/mikro-orm/mikro-orm/blob/97a9ce2af4e631c8aae8a9ee9d387bf7bc6eb5a6/packages/core/src/utils/Utils.ts#L803))

**To Reproduce**
1. Compiled config file
```javascript
"use strict";
exports.default = void 0;
const config = {
    forceUtcTimezone: true,
    forceUndefined: true,
    type: 'postgresql',
    autoLoadEntities: true
};
var _default = config;
exports.default = _default;

//# sourceMappingURL=mikro-orm.config.js.map
```
2. Run 
```
mikro-orm debug
```

**Expected behavior**
Load config file.

**Versions**

| Dependency | Version |
| - | - |
| node | 16.13.1 |
| typescript | 4.5.5 |
| mikro-orm | 5.0.2 |
| @swc/core | 1.2.141 |

**Comment by B4nan** [comment]
there is no dynamic import (not on the line you highlighted), it is still require call, check how the compiled code looks like - whole project is compiled to commonjs, and you need that `MIKRO_ORM_DYNAMIC_IMPORTS` env var to actually use dynamic import (via the other branch with `eval()`).

seeing you are using swc, that will be rather your problem. i'll be happy to look into this, but full reproduction would be required.
